release-notes.md

```diff
@@ -1,5 +1,17 @@
 # Service Broker API Release Notes #
 
+## <a id='2-12'></a>v2.12 ##
+2017-06-13
+
+* Added `context` field to request body for provisioning a service instance (`PUT /v2/service_instances/:instance_id`) and updating a service instance (`PATCH /v2/service_instances/:instance_id`). Also added the [profile.md](https://github.com/openservicebrokerapi/servicebroker/blob/master/profile.md) file describing RECOMMENDED usage patterns for environment-specific extensions and variations. `context` will replace the `organization_guid` and `space_guid` request fields in future versions of the specification. In the interim, both SHOULD be used to ensure interoperability with old and new implementations.
+* The specification now uses RFC2119 keywords to make it clearer when things are REQUIRED rather than OPTIONAL or RECOMMENDED.
+* Request and response parameters of type string, if present, MUST be a non-empty string.
+* Cleaned up text around status codes in responses, clarifying when they MUST be returned.
```

Comment:
I would make this a sublist of 'Numerous clarifications' and remove the various 'clarify' variations in each bullet